docs/source/quickstart.mdx

```diff
@@ -111,23 +102,17 @@ Next, we load an _optimizer_ to apply these optimisations to our model:
 ```python
 >>> from optimum.onnxruntime import ORTOptimizer
 
->>> optimizer = ORTOptimizer.from_pretrained(
-...     model_checkpoint,
-...     feature="sequence-classification",
-... )
+>>> optimizer = ORTOptimizer.from_pretrained(ort_model)
```

Comment:
Good catch, I fixed it and use now `ort_model` to bring clarity among the different section so that it's clear that both `ORTQuantizer` / `ORTOptimizer` should be instantiated with ORTModels.

Reply:
I was not able to reproduce the error you are describing. Did you had issues running : 

```python 
from transformers import AutoTokenizer
from optimum.onnxruntime import ORTModelForSequenceClassification, ORTOptimizer
from optimum.onnxruntime.configuration import OptimizationConfig

model_checkpoint = "distilbert-base-uncased-finetuned-sst-2-english"
save_directory = "tmp/onnx/"
# Load a model from transformers and export it through the ONNX format
ort_model = ORTModelForSequenceClassification.from_pretrained(model_checkpoint, from_transformers=True)
tokenizer = AutoTokenizer.from_pretrained(model_checkpoint)
# Save the onnx model and tokenizer
ort_model.save_pretrained(save_directory)
tokenizer.save_pretrained(save_directory)
optimizer = ORTOptimizer.from_pretrained(ort_model)
# Optimize the model
optimization_config = OptimizationConfig(optimization_level=1)
optimizer.optimize(save_dir=save_directory, optimization_config=optimization_config)
```

Reply:
This code snippet runs without any issue. It seems this error happens only when the ORT model is loaded from an existing ONNX file. Could you try with the following code snippet and let me know if you manage to reproduce it:
```python
from transformers import AutoTokenizer
from optimum.onnxruntime import ORTModelForSequenceClassification, ORTOptimizer
from optimum.onnxruntime.configuration import OptimizationConfig

model_checkpoint = "distilbert-base-uncased-finetuned-sst-2-english"
save_directory = "tmp/onnx/"
# Load a model from transformers and export it through the ONNX format
ort_model = ORTModelForSequenceClassification.from_pretrained(model_checkpoint, from_transformers=True)
tokenizer = AutoTokenizer.from_pretrained(model_checkpoint)
# Save the onnx model and tokenizer
ort_model.save_pretrained(save_directory)
tokenizer.save_pretrained(save_directory)

model = ORTModelForSequenceClassification.from_pretrained(save_directory, file_name="model.onnx")
optimizer = ORTOptimizer.from_pretrained(model)
# Optimize the model
optimization_config = OptimizationConfig(optimization_level=1)
optimizer.optimize(save_dir=save_directory, optimization_config=optimization_config)
```

The `config.json` file does not contain the `model_type` key in that case.